coilsnake/modules/eb/SpriteGroupModule.py

```diff
@@ -21,6 +21,13 @@ class SpriteGroupModule(EbModule):
                    (0x130000, 0x13ffff),
                    (0x140000, 0x14ffff),
                    (0x150000, 0x154fff)]
+    SPRITE_GROUP_TABLE_RELOCATIONS = [
+        0x001DF9,
+        0x001E79,
+        0x001FE0,
+        0x007A8B,
+        0x04B1D0,
+    ]
```

Comment:
Can you change these to `AsmPointerReference` instances instead? I think that would be nicer than needing to know they're all AsmPointers

Additionally, can you rename the attribute to `SPRITE_GROUP_TABLE_REFERENCES` to match with the new naming convention?